PurrSoft/PurrSoft.Application/Commands/AnimalCommands/AnimalCommands.cs

```diff
@@ -0,0 +1,36 @@
+﻿using PurrSoft.Application.Common;
+using PurrSoft.Domain.Entities;
+using PurrSoft.Domain.Entities.Enums;
+
+namespace PurrSoft.Application.Commands.AnimalCommands;
+
+public class AnimalGetCommand: BaseRequest<CollectionResponse<Animal>>
```

Comment:
Do not put getters in commands.
Commands are for modifications that happen in the database that modify the data.
Put it in Queries same pattern ( kind of) like here
Queries:
- AnimalQueries (with all your getters as classes)
- AnimalQueriesHandler
Contact me if you need help there :).